planet/src/main/java/com/google/planet/data/Event.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.planet.data;
+
+public final class Event {
+    private final long id;
+    private final String name; 
+    private final String address;
+    private final double duration; 
+    private final String preferredTime;
```

Comment:
Consider creating a type for time. You can use java's time: https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html

or create your own type.

(prefer strongly typed code instead of 'stringly' typed code): http://wiki.c2.com/?PrimitiveObsession

Reply:
Okay!